apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -113,8 +115,11 @@
 
 #define LONG_FILE_CONTENTS "Yesterday all my trouble seemed so far away. Now it looks as though they're here to stay. Oh, I believe in yesterday."
 
+#define DEV_RAMDISK_PATH "/dev/ram2"
+
 #define LONG_FILE_LOOP_COUNT 24
 
+
```

Comment:
@kavyasrinunna 
Empty line can be removed

Reply:
removed